AK/Format.h

```diff
@@ -423,6 +423,13 @@ struct Formatter<char> : StandardFormatter {
     void format(FormatBuilder&, char value);
 };
 template<>
+struct Formatter<wchar_t> : StandardFormatter {
+    void format(FormatBuilder& builder, wchar_t value)
+    {
+        builder.put_u64(value, 16, true);
```

Comment:
considering `wchar`s are characters maybe at least try to do an acscii- or utf-lookup

Reply:
Do you have any examples on how I would do an UTF-8 lookup (ASCII I can probably manage myself)? Also, I don't see this being used much apart from tests, and for test things I'd actually prefer printing it as hexadecimal.

Reply:
I am sadly not very knowledgeable with whchars, I only know that "they can address a bigger character set", but i do not know which "bigger" char-set they address here.
If you know that maybe make an Array, where you lookup the wchar and get the utf-8 code points for it

Reply:
The way that `wchar_t` is implemented right now is that one `wchar_t` contains one code point (i.e. the up to 21 bytes that are stored in a single UTF-8 sequence). So I'd basically just need a way of telling the system "here is a code point, now print it as a unicode character".

Reply:
StringBuilder has `append_code_point()` if that's helpful.

Reply:
Yeah, that seems to work! I'll push a new revision using that.